blues/tests/test_mc.py

```diff
@@ -0,0 +1,128 @@
+
+import unittest, parmed
+import numpy as np
+from openmmtools.testsystems import get_data_filename
+from blues.moves import RandomLigandRotationMove
+from blues.engine import MoveEngine
+from blues.simulation import Simulation, SimulationFactory
+from simtk import openmm
+from openmmtools import testsystems
+import types
+import math
+
+class BLUESTester(unittest.TestCase):
+    """
+    Test the Simulation class.
+    """
+    def setUp(self):
```

Comment:
Do you want to reduce the redundancy between this and the `test_blues` module by, maybe, putting them into the same test script and then making a function both can call that does some of the setup common to both? I'm just wondering if there's an easy way to reduce redundancy so there is not as much to update when we want to change something in these tests. Your call on this one.